sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CoderRegistryTest.java

```diff
@@ -89,8 +90,14 @@ public void testSerializableFallbackCoderProvider() throws Exception {
   public void testProtoCoderFallbackCoderProvider() throws Exception {
     CoderRegistry registry = getStandardRegistry();
 
+    //must use reflection here or a reference to dataflow class is baked into
```

Comment:
No, whether I specify a profile or not, only the two tests marked with the Category are run.   However, when the test is compiled with edt, the byte code is different enough that calls to the CoderRegistryTest.class.getName() and several other things that that are done internally will throw an exception as it tries to completely validate the entire class and the dataflow class isn't available.   I don't know why it works when compiled with javac but not with edt.